apps/jobs/sender.py

```diff
@@ -90,6 +90,8 @@ def publish_submission_message(message):
     # push metrics to pushgateway as a submission is pushed into queue
     job_id = "submission_{}".format(submission_pk)
     push_metrics_to_pushgateway(job_id)
+    # clear metrics after they are pushed to avoid duplication on next push
+    num_submissions_in_queue.clear()
```

Comment:
Why do we have to clear the metrics explicitly? I have used prometheus metrics we never had to clear the metrics.